docs(config options): rewrite `forkProcessing` (#21621)
renovatebot/renovate: docs/usage/configuration-options.md

## forkProcessing

By default, Renovate skips any forked repositories when in `autodiscover` mode.
It even skips a forked repository that has a Renovate configuration file, because Renovate doesn't know if that file was added by the forked repository.

**Process a fork in `autodiscover` mode`**

If you want Renovate to run on a forked repository when in `autodiscover` mode then:

> - Add `"forkProcessing": "enabled"` to your repository config,

Reviewer: ```suggestion
- Ensure a `renovate.json` config exists with `"forkProcessing": "enabled"` in your repository,
```

- Or run the CLI command with `--fork-processing=enabled`

### Process a fork in other modes

> If you're running Renovate in some other mode, for example when giving a list of repositories to Renovate, but want to skip forked repositories: set `"forkProcessing": "disabled"` in your _global_ config.

Author: I'm out of ideas on how to rewrite this to be shorter or spread over two sentences. Can you suggest a better wording?

> If you're running Renovate in some other mode, for example when giving a list of repositories to Renovate, but want to skip forked repositories: you must configure `"forkProcessing": "disabled"` in your global config.

Author: ```suggestion
If you're running Renovate in some other mode, for example when giving a list of repositories to Renovate, but want to skip forked repositories: set `"forkProcessing": "disabled"` in your _global_ config.
```
